sample-app/src/components/DropdownSection.tsx

```diff
@@ -87,7 +87,7 @@ export default function DropdownSection({
 
   function renderOption(option: Option, index: number) {
     const className = classNames(cssClasses.option, {
-      [cssClasses.focusedOption]: focusStatus === 'active' && index === focusedOptionIndex
+      [cssClasses.focusedOption ?? '']: focusStatus === 'active' && index === focusedOptionIndex
```

Comment:
Could we change this logic? It's a minor nit, but having 
```
{ '': true/false }
```
feels kinda odd. Maybe we don't have this boolean check unless `focusedOption` is defined.

Reply:
I made an update which addresses the nit, however I feel like it's missing the point of the `classNames` library, but let me know if you prefer it